build/docker/mysql.conf.d/custom.cnf

```diff
@@ -0,0 +1,20 @@
+[mysqld]
+bind-address = 0.0.0.0
+
+default-authentication-plugin=mysql_native_password
```

Comment:
This is necessary. But from where are the others coming from? Are they all from the mysql.cnf in the server repo?

Reply:
Basically yes, wanted to have same configuration.